src/Sulu/Component/SmartContent/Tests/Unit/DataProviderPoolTest.php

```diff
@@ -175,19 +175,21 @@ public static function getProvider()
         ];
     }
 
-    /**
-     * @dataProvider getProvider
-     */
-    public function testGet(DataProviderPool $pool, $alias, $expectedProvider, $exceptionName = null): void
-    {
+    #[\PHPUnit\Framework\Attributes\DataProvider('getProvider')]
+    public function testGet(
+        DataProviderPool $pool,
+        string $alias,
+        ?DataProviderInterface $expectedProvider,
+        ?string $exceptionName = null,
+    ): void {
```

Comment:
should be done on 2.5 branch:

```suggestion
    public function testGet(DataProviderPool $pool, string $alias, ?DataProviderInterface $expectedProvider, ?string $exceptionName = null): void {
```